### ISSUE TITLE
increased tubuloglomerular feedback response

### ISSUE THREAD

**Issue by slaulederkind** [mgijax/mammalian-phenotype-ontology#3584]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)

increased tubuloglomerular feedback response

>2. Synonyms (e.g., absent spleen)

improved tubuloglomerular feedback response

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

increase of the blood flow control mechanism that is operated in the juxtaglomerular apparatus of the kidney and involved in renal autoregulation by limiting changes in the glomerular filtration rate. TGF responds to changes in the concentration of NaCl in the tubular fluid at the level of the macula densa.

PMID:31932071

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

abnormal tubuloglomerular feedback response (MP:0011295)

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

ORCID:0000-0001-5256-8683

**Comment by anna-anagnostop** [comment]
Added new term:
increased tubuloglomerular feedback response MP:0031404
DEF: enhancement of the blood flow control mechanism that is operated in the juxtaglomerular apparatus of the kidney and involved in renal autoregulation by limiting changes in the glomerular filtration rate. TGF responds to changes in the concentration of NaCl in the tubular fluid at the level of the macula densa
SYN: improved tubuloglomerular feedback response
SYN: improved TGF response
SYN: enhanced tubuloglomerular feedback response
SYN: enhanced TGF response
SYN: increased TGF response
